### ISSUE TITLE
ie11 legend positioning bug

### ISSUE THREAD

**Issue by akvanhar** [recharts/recharts#364]
I have the following legend code:
`<Legend
    align='right'
    verticalAlign='top'
    height=36
    iconSize=10
/>`

This works as expected in Chrome, Safari, Firefox, MSEdge
But in ie11, the positioning is wrong.

My workaround: to add custom css top: -60px specifically for ie11

See attached screen shots.
![ie11legendbug](https://cloud.githubusercontent.com/assets/10929533/20576130/69648c22-b171-11e6-980d-598278fa61b3.jpg)
![chromelegendnobug](https://cloud.githubusercontent.com/assets/10929533/20576136/70ea8fa0-b171-11e6-9e0f-e3427118022e.jpg)

**Comment by vaeum** [comment]
Hello, add this css code in your project

```css
.recharts-legend-item .recharts-surface{
  height: ${your icon height in pixels};
}
```

**Comment by akvanhar** [comment]
Thanks. I was doing complicated things with ie-11 browser selection. I prefer this browser agnostic solution.

**Comment by xile611** [comment]
@akvanhar I think this has been fixed in version 0.20.0. Please check it!
